src/docs/testing/integration-tests/index.md

```diff
@@ -45,38 +45,26 @@ or desktop application, called the **target** device.
 browser or desktop application,
 the host machine is also the target device.)
 
-**flutter_driver** 
-
-The `flutter_driver` package runs integration tests
-written in Dart on a target device and reports the
-results to the host. Tests written with `flutter_driver`
-run from the host and drive the app running on a real
-or virtual device. The `flutter drive` command is used
-to run tests written with this package.
-
 **integration_test**
 
 Tests written with the `integration_test` package can:
  
 1. Run directly on the target device, allowing you to test on
    multiple Android or iOS devices using Firebase Test Lab.
-2. Run using `flutter_driver`. 
+2. Run using `flutter test`. 
```

Comment:
Perhaps `flutter test integration_test` to avoid confusion with `flutter test` 🙂